src/estimators/tests/any_sketch_test.py

```diff
@@ -37,6 +38,29 @@ def test_uniform_distribution(self):
     self.assertEqual(u1, u2)
     self.assertEqual(u1.get_index(42), 2)
 
+  def test_geometric_distribution(self):
+    u1 = GeometricDistribution(10, 0.08)
+    u2 = GeometricDistribution(10,)
+    self.assertLen(u1, 10)
+    self.assertEqual(u1, u2)
+    self.assertEqual(u1.get_index(42), 0)
+
+  def test_geo_bucket_distribution_get_bounds(self):
+    bounds = GeometricDistribution(num_values=4, probability=0.08).register_bounds
+    expected = np.array([0.28208044, 0.54159445, 0.78034734, 1.])
+    np.testing.assert_allclose(bounds, expected, atol=1e-8)
+
+  @parameterized.parameters(
+      (0, 0),
+      (0.28208044, 1),
+      (0.5, 1),
+      (0.8, 3),
+      (0.94, 3),
+      (1, 3))
+  def test_log_bucket_distribution_get_index(self, hash_value, index):
+    exp_dist = LogBucketDistribution(num_values=4, probability=0.08)
```

Comment:
Copy paste error?

Reply:
Same as any_sketch.py where the file was left in a wrong state - I'll double check diff view in the future. This is basically the parameterized test on GeometricDistribution